src/Resources/contao/templates/backend/be_main.html5

```diff
@@ -54,23 +54,7 @@
   <div id="container" class="cf">
     <?php if (!$this->isPopup): ?>
       <div id="left">
-        <nav id="tl_navigation" role="navigation">
```

Comment:
If we would keep this element here and just render the `ul` part, we could skip the skipNavigation part from our menu renderer (including the language labels in Twig 😉 )